app/routes/picker.js

```diff
@@ -0,0 +1,12 @@
+const { GET } = require('../constants/http-verbs')
+const { USER } = require('ffc-auth/scopes')
+const { serverConfig } = require('../config/index')
+
+module.exports = [{
+  method: GET,
+  path: '/picker',
+  options: { auth: { strategy: 'jwt', scope: [USER] } },
+  handler: async (request, h) => {
+    return h.redirect(`${serverConfig.gatewayHost}/auth/picker`)
```

Comment:
Consider if this route should have a `redirect` query param and where should we be redirecting them? 
Cannot use the existing pattern from `errors.js` as this creates an endless loop of going back to the picker!

Currently the auth repo sets the redirect to the home page which is probably fine for now